src/hub.js

```diff
@@ -195,10 +195,16 @@ const onReady = async () => {
     if (NAF.connection.adapter && NAF.connection.adapter.localMediaStream) {
       NAF.connection.adapter.localMediaStream.getTracks().forEach(t => t.stop());
```

Comment:
does this cover clean up of `mediaStream`?

Reply:
mediaStreams should be cleaned up by `networked-aframe` when the a-scene is removed from the DOM